src/etaoin/api.clj

```diff
@@ -3546,9 +3547,29 @@
                          (drv/set-capabilities (:capabilities defaults-global))
                          (drv/set-capabilities (get-in defaults [type :capabilities]))
                          (drv/set-capabilities capabilities)))
-        caps          (:capabilities driver)
-        session       (create-session driver caps)]
-    (assoc driver :session session)))
+        caps          (:capabilities driver)]
+    (loop [n 1]
+      ;; Wait for driver to be ready before creating the first session.
```

Comment:
I guess we'll retry even if the driver process is no longer running (has crashed/exited). This is unlikely, so probably fine.

Reply:
Correct. At this point, we believe the problem is with the session state (either in the driver or in the browser; I'm still not sure), not that the driver has crashed.